spark/src/main/scala/geotrellis/spark/tiling/MapKeyTransform.scala

```diff
@@ -95,12 +104,15 @@ class MapKeyTransform(val extent: Extent, val layoutCols: Int, val layoutRows: I
     (tcol.floor.toInt, trow.floor.toInt)
   }
 
-  def apply[K: SpatialComponent](key: K): Extent = {
-    apply(key.getComponent[SpatialKey])
-  }
+  @deprecated("Use this instead: MapKeyTransform.keyToExtent(key.getComponent[SpatialKey])", "1.2")
+  def apply[K: SpatialComponent](key: K): Extent = apply(key.getComponent[SpatialKey])
+
+  def keyToExtent(key: SpatialKey): Extent = apply(key)
+
+  def apply(key: SpatialKey): Extent = apply(key.col, key.row)
 
-  def apply(key: SpatialKey): Extent =
-    apply(key.col, key.row)
+  /** 'col' and 'row' correspond to a [[SpatialKey]] column and row in some grid. */
+  def coordsToExtent(col: Int, row: Int): Extent = apply(col, row)
```

Comment:
Can this be `keyToExtent` overload? `coords` is a little overloaded, is it pixel cords, tile cords, map cords, a bit of ambiguity.

Reply:
I don't think those are the same thing. See the `apply` that it calls out to right below it.

Reply:
Oh wait, I might be dumb.